docs/appendices/types.rst

```diff
@@ -57,6 +57,17 @@ records.
 
 .. _types-caa:
 
+APL
```

Comment:
missing `.. _types-apl:`

Reply:
Fixed in https://github.com/PowerDNS/pdns/pull/8638/commits/2bc5a6dc358cdf54aae774df2fc3552ebd3de8bc

Reply:
it should go before the apl title, and caa should be before caa title :)